dataland-frontend/src/components/forms/CreateEuTaxonomyForNonFinancials.vue

```diff
@@ -389,7 +391,7 @@ export default defineComponent({
       type: String,
     },
   },
-  created() {
+  mounted() {
```

Comment:
Why has this changed to `mounted` ?
In my opinion, `created` is better, because then it already starts fetching the data at an earlier stage of the lifecycle.

Reply:
@pioSko2 @MarcinStanczyk7 what was the reason for the change?

Reply:
No idea. Sorry.

Reply:
Changed it to created(). If there are objections we can change it back

Reply:
Ok.